MyBike layout

ShamievDavid opened the pull request:
[DEMO LINK](https://ShamievDavid.github.io/layout_miami/)

IvanFesenko reviewed (changes requested):
Well done - keep going

etojeDenys reviewed (changes requested):
fix this mismatch:
<img width="335" alt="image" src="https://github.com/mate-academy/layout_miami/assets/70220667/3ece140e-8159-4f43-8615-9f477bdc2635">
<img width="368" alt="image" src="https://github.com/mate-academy/layout_miami/assets/70220667/f028a272-151d-46f2-a8ee-cc3e8472cd89">
2. and add hover to photos

ShamievDavid commented:
For now i added 'details block' without images. Because I didn't find solution, so i will go to Q&A with that question tommorow.

mykhalenych reviewed (changes requested):
https://github.com/mate-academy/layout_miami/assets/53228259/0e265b29-87ee-49a4-bdfa-8f32a2fee307

1. fix scroll issue

<img width="331" alt="Screenshot 2023-05-22 at 16 11 50" src="https://github.com/mate-academy/layout_miami/assets/53228259/2bc5e9d6-098e-49e9-a9ad-472c7592da39">
2. add hover to links
3. add validation to inputs

ShamievDavid commented:
1. i fixed scroll issue 
2. i added on hover (there is no design, so i just made color "yellow" on hover. 
3. i don't really understood, what do you mean by validation in this case? 
I made every field "required" but previously i turn off default behavior of the form, because I saw it in checklist. 
so now when you click the submit button - happens nothing. Maybe i should use event.preventDefault in another way? 
Or what do you mean by validation? Thank you.

VitaliyBondarenko1982 reviewed (changes requested):
Almost done.
Form still reload the page 
![Screenshot from 2023-05-23 15-44-55](https://github.com/mate-academy/layout_miami/assets/33029864/d1ed3072-4568-42e6-ba3d-048c3871c71d)
To fix it add such code to main.js

```
  const form = document.querySelector('form');

form.addEventListener('submit', (event) => {
  event.preventDefault();

  form.reset();
})

VitaliyBondarenko1982 reviewed (approved):
Looks good to me.

VitaliyBondarenko1982 reviewed (approved):
Looks good to me.
